editor/inserter/menu.js

```diff
@@ -225,7 +225,15 @@ class InserterMenu extends wp.element.Component {
 	render() {
 		const { position = 'top', instanceId } = this.props;
 		const visibleBlocksByCategory = this.getVisibleBlocksByCategory( wp.blocks.getBlockTypes() );
-		const positionClasses = position.split( ' ' ).map( ( pos ) => `is-${ pos }` );
+		const visualEditorHeight = document.querySelector( '.editor-visual-editor' ).clientHeight;
+		const minimumNeededHeight = 600;
```

Comment:
Yes, I think so. Or even `clientHeight` or `getClientBoundingRect` could work just as well.

It's not possible to do so here in the `render` though. Since the first time it would be called, there is not an actual DOM node yet (`render` only returns the React element). Instead, you'd need a combination of [`ref`](https://facebook.github.io/react/docs/refs-and-the-dom.html) and `componentDidMount` (and more, per https://github.com/WordPress/gutenberg/pull/1193#issuecomment-308719451).